parser/model/model.go

```diff
@@ -448,6 +448,11 @@ type TableInfo struct {
 
 	// StatsOptions is used when do analyze/auto-analyze for each table
 	StatsOptions *StatsOptions `json:"stats_options"`
+
+	// ExchangePartitionFlag、ExchangePartitionId、ExchangePartitionDefId is used when do exchange partition with table
```

Comment:
Please wrap the 3 new fields into 1 struct, for example:

```
type ExchangePartitionInfo struct {
	ExchangePartitionFlag  bool  `json:"exchange_partition_flag"`
	ExchangePartitionId    int64 `json:"exchange_partition_id"`
	ExchangePartitionDefId int64 `json:"exchange_partition_def_id"`
}
```

Reply:
done.